### PR TITLE
feat: video speech transcription

### PR DESCRIPTION
**Pull request by anguillanneuf**

### CODE REVIEW

File: video/cloud-client/src/test/java/com/example/video/DetectIT.java [hunk 1]
```diff
@@ -37,6 +37,7 @@
   static final String LABEL_FILE_LOCATION = "./resources/cat.mp4";
   static final String SHOTS_FILE_LOCATION = "gs://demomaker/gbikes_dinosaur.mp4";
   static final String EXPLICIT_CONTENT_LOCATION =  "gs://demomaker/cat.mp4";
+  static final String SPEECH_GCS_LOCATION = "gs://python-docs-samples-tests/video/googlework_short.mp4";
```
Comment (nnegrey) [inline video/cloud-client/src/test/java/com/example/video/DetectIT.java]: nit: is this also in the java-docs-samples GCS bucket?

File: video/cloud-client/src/main/java/com/example/video/Detect.java [hunk 2]
```diff
@@ -322,4 +332,69 @@ public static void analyzeExplicitContent(String gcsUri) throws Exception {
...
+              }
+            } else {
+              System.out.println("No transcription found");
+            }
+          } catch (IndexOutOfBoundsException ioe) {
+            System.out.println("Could not retrieve frame: " + ioe.getMessage());
+          }
+        }
+      }
+      // [END video_speech_transcription_gcs]
```
[the remark this answers is not in the document]
Comment (anguillanneuf) [reply video/cloud-client/src/main/java/com/example/video/Detect.java]: Right, it's been moved outside the bracket now.

File: video/cloud-client/src/test/java/com/example/video/DetectIT.java (hunk 1 above)
Comment (anguillanneuf) [reply video/cloud-client/src/test/java/com/example/video/DetectIT.java]: Yes, it is there.

File: video/cloud-client/src/main/java/com/example/video/Detect.java [hunk 3]
```diff
@@ -322,4 +332,69 @@ public static void analyzeExplicitContent(String gcsUri) throws Exception {
...
+
+      // Set the video context with the above configuration
+      VideoContext context = VideoContext.newBuilder()
+              .setSpeechTranscriptionConfig(config)
+              .build();
+
+      // Create the request
+      AnnotateVideoRequest request = AnnotateVideoRequest.newBuilder()
+              .setInputUri(gcsUri)
+              .addFeatures(com.google.cloud.videointelligence.v1.Feature.SPEECH_TRANSCRIPTION)
```
[the remark this answers is not in the document]
Comment (anguillanneuf) [reply video/cloud-client/src/main/java/com/example/video/Detect.java]: Fixed.

**Pull request merged**